po/pl.po

```diff
@@ -109,297 +117,336 @@ msgstr "Twórz obrazy gotowe do publikacji"
+#: gradia/ui/image_loaders.py:222
 msgid "Failed to load image from clipboard."
 msgstr "Błąd podczas ładowania obrazu ze schowka."
 
-#: gradia/ui/image_loaders.py:206 gradia/ui/image_loaders.py:221
+#: gradia/ui/image_loaders.py:255 gradia/ui/image_loaders.py:270
 msgid "Failed to take screenshot"
 msgstr "Błąd wykonywania zrzutu ekranu"
 
-#: gradia/ui/image_loaders.py:237
+#: gradia/ui/image_loaders.py:286
 msgid "Screenshot cancelled"
 msgstr "Anulowano wykonywanie zrzutu ekranu"
 
-#: gradia/ui/image_loaders.py:253
+#: gradia/ui/image_loaders.py:302 gradia/ui/image_loaders.py:328
 msgid "Edited Screenshot From %Y-%m-%d %H-%M-%S"
-msgstr "Edycja zrzutu ekranu od %Y-%m-%d %H-%M-%S"
+msgstr "Edytowane zrzutu ekranu z %Y-%m-%d %H-%M-%S"
 
-#: gradia/ui/image_loaders.py:259 gradia/ui/image_loaders.py:260
+#: gradia/ui/image_loaders.py:308 gradia/ui/image_loaders.py:309
+#: gradia/ui/image_loaders.py:335 gradia/ui/image_loaders.py:336
 msgid "Screenshot"
 msgstr "Zrzut ekranu"
 
-#: gradia/ui/image_loaders.py:263
+#: gradia/ui/image_loaders.py:312 gradia/ui/image_loaders.py:340
 msgid "Screenshot captured!"
 msgstr "Przechwycono zrzut ekranu!"
 
-#: gradia/ui/image_loaders.py:270
+#: gradia/ui/image_loaders.py:319
 msgid "Failed to process screenshot"
 msgstr "Błąd przetwarzania zrzutu ekranu"
 
-#: gradia/ui/image_loaders.py:341
+#: gradia/ui/image_loaders.py:344
+msgid "Failed to load screenshot"
+msgstr "Błąd ładowania zrzutu ekranu"
+
+#: gradia/ui/image_loaders.py:413
 msgid "Generated Image"
 msgstr "Wygenerowany obraz"
 
-#: gradia/ui/image_loaders.py:342
+#: gradia/ui/image_loaders.py:414
 msgid "Source"
 msgstr "Źródło"
 
-#: gradia/ui/image_loaders.py:345
+#: gradia/ui/image_loaders.py:417
 msgid "Source snippet Generated!"
 msgstr "Wygenerowano fragmenty źródłowe!"
 
-#: gradia/ui/provider_selection_window.py:38
-#: gradia/ui/provider_selection_window.py:133
+#: gradia/ui/provider_selection_window.py:37
 msgid "Choose Provider"
 msgstr "Wybierz dostawcę"
 
-#: gradia/ui/provider_selection_window.py:70
+#: gradia/ui/provider_selection_window.py:66
 msgid "Loading providers data..."
 msgstr "Ładowanie danych dostawcy..."
 
-#: gradia/ui/provider_selection_window.py:85
+#: gradia/ui/provider_selection_window.py:81
 msgid "An error occurred"
 msgstr "Wystąpił problem"
 
-#: gradia/ui/provider_selection_window.py:86
+#: gradia/ui/provider_selection_window.py:82
 msgid "Please try again later."
 msgstr "Spróbuj ponownie później."
 
-#: gradia/ui/provider_selection_window.py:166
-#: gradia/ui/provider_selection_window.py:200
+#: gradia/ui/provider_selection_window.py:147
+#: gradia/ui/provider_selection_window.py:236
 msgid "Custom Provider"
 msgstr "Własny dostawca"
 
-#: gradia/ui/provider_selection_window.py:167
+#: gradia/ui/provider_selection_window.py:148
 msgid "Create your own custom upload command"
-msgstr "Stwórz swoją własną komendę do wczytania"
+msgstr "Stwórz swoją własną komendę do przesłania"
```

Comment:
The command isn't created to be send, it is created to send something, so "przesłać" at the end should be inflected as "przesyłania" and ideally the "do" preposition should be removed from sentence.